src/api/app/views/webui/code_of_conduct/index.html.haml

```diff
@@ -0,0 +1,6 @@
+- @pagetitle = 'Code of Conduct'
+.card
+  .card-body
+    %h3.mb-4 Code of Conduct
+    %textarea.form-control.font-monospace.code-of-conduct{ disabled: true }
+      = @code_of_conduct
```

Comment:
How about markdown? :)

Reply:
I switched it to markdown.

Reply:
Hm sorry  I don't think markdown is an option as I just remembered that `Configuration` has an API and markdown in XML is not possible AFAIR...

Reply:
In the end I changed it to use HTML, for consistency with the `description` field. Please take a look again.